specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-01-01/machineLearningServices.json

```diff
@@ -1198,7 +1199,7 @@
           }
         ],
         "responses": {
-          "200": {
+          "202": {
             "description": "Success."
```

Comment:
Could we also update the description?

Reply:
To keep it consistent with start and stop, I have kept the description same.